src/main/java/cms/gov/madie/measure/resources/MeasureTransferController.java

```diff
@@ -173,6 +171,29 @@ private void updateOrganizationName(Measure measure) {
     }
   }
 
+  /**
+   * @param measure mat measure There may have references. These references need to have a UUID
+   *     attached to them on transfer.
+   */
+  void updateReferenceListIds(Measure measure) {
+    List<Reference> referenceList = measure.getMeasureMetaData().getReferences();
```

Comment:
Any particular reason we are doing this in measure-service instead of transfer-lambda-service ?

There are couple of other fields ( I know measureDefinitions is one of them ) which also required uuid and I believe we are generating them in lambda service itself.

Reply:
https://github.com/MeasureAuthoringTool/measure-transfer-lambda/compare/MAT-6490?expand=1

lambda approach made much more sense